### PR TITLE
DOC-9770 Update Observability docs for Cloud 2.0 - Export Metrics and Essential Metrics 

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-9770

(1) In export-metrics-advanced.md, replaced dedicated variable with advanced and fixed/added links. (2) In tools-page.md, in header replaced dedicated variable with cloud. (3) In export-metrics.md, fixed/added links. (4) In other files, fixed links to export-metrics, export-metrics-advanced and tools-page.

(1) In essential-metrics.md include file, replaced (a) 66 instances of include.deployment == 'dedicated' with  include.deployment == 'advanced', (b) 11 instances of crdb_dedicated with crdb_cloud (see Jira [CC-26604](https://cockroachlabs.atlassian.net/browse/CC-26604)), (c) KEPT 11 instances of cockroach_dedicated in https://docs.datadoghq.com/integrations/cockroachdb_dedicated. (2) Renamed essential-metrics-dedicated.md to essential-metrics-advanced.md. (3) In other files, fixed links to essential-metrics-advanced.

### CODE REVIEW

File: src/current/_includes/v23.1/essential-metrics.md [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
 {% if include.deployment == 'self-hosted' %}
 * [Grafana]({% link {{ page.version.version }}/monitor-cockroachdb-with-prometheus.md %}#step-5-visualize-metrics-in-grafana)
 * [Datadog Integration]({% link {{ page.version.version }}/datadog.md %}) - The [**Datadog Integration Metric Name**](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics) column lists the corresponding Datadog metric which requires the `cockroachdb.` prefix.
-{% elsif include.deployment == 'dedicated' %}
-* [Datadog integration]({% link cockroachcloud/tools-page.md %}#monitor-cockroachdb-dedicated-with-datadog) - The [**Datadog Integration Metric Name**](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics) column lists the corresponding Datadog metric which requires the `crdb_dedicated.` prefix.
+{% elsif include.deployment == 'advanced' %}
+* [Datadog integration]({% link cockroachcloud/tools-page.md %}#monitor-cockroachdb-cloud-with-datadog) - The [**Datadog Integration Metric Name**](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics) column lists the corresponding Datadog metric which requires the `crdb_cloud.` prefix.
```
Comment (rmloveland) [inline src/current/_includes/v23.1/essential-metrics.md:6]: is this change to the prefix from `crdb_dedicated` to `crdb_cloud` something that exists now or planned work?  I assume there are code changes required .  Is there an engineering issue / PR about this you could share?

File: src/current/_includes/v23.1/essential-metrics.md [hunk 2]
```diff
@@ -1,54 +1,54 @@
...
-| ----------------------------------------------------- | {% if include.deployment == 'self-hosted' %}------ |{% elsif include.deployment == 'dedicated' %}---- |{% endif %} ------------------------------------------------------------ | ------------------------------------------------------------ |
+|<div style="width:225px">CockroachDB Metric Name</div> | {% if include.deployment == 'self-hosted' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics)<br>(add `cockroachdb.` prefix)</div> |{% elsif include.deployment == 'advanced' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics)<br>(add `crdb_cloud.` prefix)</div> |{% endif %}<div style="width:150px">Description</div>| Usage |
```
Comment (rmloveland) [inline src/current/_includes/v23.1/essential-metrics.md:14]: same question re: `crdb_dedicated` change to `crdb_cloud`

File: src/current/_includes/v23.1/essential-metrics.md [hunk 3]
```diff
@@ -1,54 +1,54 @@
...
-| capacity                                            | {% if include.deployment == 'self-hosted' %}capacity.total  |{% elsif include.deployment == 'dedicated' %}capacity |{% endif %} Total storage capacity                                       | This metric gives total storage capacity. Measurements should comply with the following rule: CockroachDB storage volumes should not be utilized more than 60% (40% free space). |
+<div style="width:225px">CockroachDB Metric Name</div> | {% if include.deployment == 'self-hosted' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics)<br>(add `cockroachdb.` prefix)</div> |{% elsif include.deployment == 'advanced' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics)<br>(add `crdb_cloud.` prefix)</div> |{% endif %}<div style="width:150px">Description</div>| Usage |
```
Comment (rmloveland) [inline src/current/_includes/v23.1/essential-metrics.md:33]: same question re: `crdb_dedicated` change to `crdb_cloud`

File: src/current/_includes/v23.1/essential-metrics.md [hunk 4]
```diff
@@ -1,54 +1,54 @@
...
-| ----------------------------------------------------- | {% if include.deployment == 'self-hosted' %}------ |{% elsif include.deployment == 'dedicated' %}---- |{% endif %} ------------------------------------------------------------ | ------------------------------------------------------------ |
+| <div style="width:225px">CockroachDB Metric Name</div> | {% if include.deployment == 'self-hosted' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics)<br>(add `cockroachdb.` prefix)</div> |{% elsif include.deployment == 'advanced' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics)<br>(add `crdb_cloud.` prefix)</div> |{% endif %}<div style="width:150px">Description</div>| Usage |
```
Comment (rmloveland) [inline src/current/_includes/v23.1/essential-metrics.md:45]: same question re: `crdb_dedicated` change to `crdb_cloud`

File: src/current/_includes/v23.1/essential-metrics.md [hunk 5]
```diff
@@ -66,38 +66,38 @@ The **Usage** column explains why each metric is important to visualize in a cus
...
-| ----------------------------------------------------- | {% if include.deployment == 'self-hosted' %}------ |{% elsif include.deployment == 'dedicated' %}---- |{% endif %} ------------------------------------------------------------ | ------------------------------------------------------------ |
-| liveness.heartbeatlatency                           | {% if include.deployment == 'self-hosted' %}liveness.heartbeatlatency-p90 |{% elsif include.deployment == 'dedicated' %}liveness.heartbeatlatency |{% endif %} Node liveness heartbeat latency                              | If this metric exceeds 1 second, it is a sign of cluster instability. |
+| <div style="width:225px">CockroachDB Metric Name</div> | {% if include.deployment == 'self-hosted' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb/?tab=host#metrics)<br>(add `cockroachdb.` prefix)</div> |{% elsif include.deployment == 'advanced' %}<div style="width:225px">[Datadog Integration Metric Name](https://docs.datadoghq.com/integrations/cockroachdb_dedicated/#metrics)<br>(add `crdb_cloud.` prefix)</div> |{% endif %}<div style="width:150px">Description</div>| Usage |
```
Comment (rmloveland) [inline src/current/_includes/v23.1/essential-metrics.md:69]: same question re: `crdb_dedicated` change to `crdb_cloud`

Comment (rmloveland) [reply src/current/_includes/v23.1/essential-metrics.md:69]: PS I see there are a lot more mentions of this prefix change so I'll stop adding comments about it now :-)

**Review by rmloveland — APPROVED** [review]
LGTM I just had one question about a prefix name change that looks like it would have to be associated with a code change somewhere.  If that is the case it would be good to add a link here for reference to the Jira issue / etc where that work happened or is happening

File: src/current/_includes/v23.1/essential-metrics.md (hunk 2 above)
Comment (florence-crl) [reply src/current/_includes/v23.1/essential-metrics.md:14]: In essential-metrics.md include file, replaced 11 instances of `crdb_dedicated` with `crdb_cloud` (see Jira [CC-26604](https://cockroachlabs.atlassian.net/browse/CC-26604)).

File: src/current/_includes/v23.1/essential-metrics.md (hunk 3 above)
Comment (florence-crl) [reply src/current/_includes/v23.1/essential-metrics.md:33]: In essential-metrics.md include file, replaced 11 instances of `crdb_dedicated` with `crdb_cloud` (see Jira [CC-26604](https://cockroachlabs.atlassian.net/browse/CC-26604)).

File: src/current/_includes/v23.1/essential-metrics.md (hunk 4 above)
Comment (florence-crl) [reply src/current/_includes/v23.1/essential-metrics.md:45]: In essential-metrics.md include file, replaced 11 instances of `crdb_dedicated` with `crdb_cloud` (see Jira [CC-26604](https://cockroachlabs.atlassian.net/browse/CC-26604)).

**Comment by rmloveland** [comment]
thanks for dropping a link to that CC issue!

**Pull request merged**